### PR TITLE
Quiz results

### PR DESCRIPTION
**Pull request by Thomson42**
Sorry if it's a bit late the commit history will show the rest of the actual code written was on time

### CODE REVIEW

**Comment by Thomson42** [comment]
My test never was able to succeed on the local test so Travis will be the same.

File: .traavis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
```
Comment (martypdx) [inline .traavis.yml]: file name has one to many `a`'s, which is why travis didn't run tests

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const source = require('../src/source');
+var assert = require('assert');
+
+describe('map', () => {
+    it('Creates a new array changing elements that did not pass the test', ()=> {
+        var array = [1, 4, 5, 7]
+        const maped = source.map(array, function callback(item, i){
+           newItem = item * item 
+           if (newItem > 20) return newItem
```
Comment (martypdx) [inline test/test.js]: This code needs to be part of the actual module, not the test

File: src/source.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+function map (array, callback) {
+    const mapped = [];
+     for(let i = 0; i < array.length; i++) {
+        mapped[i] = callback(array[i], i);
+    }
+    return mapped;
+}
```
Comment (martypdx) [inline src/source.js]: This function needs to do the work of mapping and filtering, not delegate out to the test.

File: .traavis.yml (hunk 1 above)
Comment (Thomson42) [inline .traavis.yml]: Is there a way jsLint can help check these things?

Comment (martypdx) [inline .traavis.yml]: hmm, search vs code extensions for `yaml linter`?

**Pull request merged**